lighthouse-core/computed/metrics/cumulative-layout-shift.js

```diff
@@ -40,7 +40,7 @@ class CumulativeLayoutShift {
       };
     }
 
-    const cumulativeLayoutShift =
+    let cumulativeLayoutShift =
       finalLayoutShift.args &&
       finalLayoutShift.args.data &&
       finalLayoutShift.args.data.cumulative_score;
```

Comment:
I only noticed now that cumulative_score already includes the had_recent_input exclusion. I didn't entirely expect that.
https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/renderer/core/layout/layout_shift_tracker.cc;l=405-406;drc=d75ade758b6fc32f29fa5a6b28f504024942dd7d

Instead of taking the `cumulative_score` from the last event, we could instead sum up the `score`s from each layoutShift ourselves (with the input exclusion (and this first-shift consideration))

The plus side is we would't break if the definition of cumulative_score in the trace event changes in the future. (And it's [how wv.s does it](https://github.com/GoogleChrome/web-vitals/blob/master/src/getCLS.ts))

Reply:
I don't think this PR is the place to make such a change (in a followup, sure).

It's not clear to me how these trace events could change in the future such that taking the `cumulative_score` of the last event wouldn't always be correct-ish (-ish, because of the reasons this PR exists). As an example: imagine that something is added that triggers an invalidation of CLS–either a single component shift or the entire value. Unless a shift event is emitted that provides a negative score value, a manual summing would be incorrect.

Using the last event allows Chromium to definitively say "this is CLS", and for us to easily (-ish ...) get that CLS.